lib/puppet/pops/types/type_asserter.rb

```diff
@@ -1,6 +1,7 @@
 # Utility module for type assertion
 #
-module Puppet::Pops::Types::TypeAsserter
+module Puppet::Pops::Types
+module TypeAsserter
```

Comment:
It might seem odd at first but this construct allows everything in the TypeAsserter module to reference everything in the Puppet::Pops::Types module without qualification. It has nothing to do with module initialization.